test/integration/etcd_storage_path_test.go

```diff
@@ -425,7 +433,14 @@ var etcdStorageData = map[schema.GroupVersionResource]struct {
 	gvr("autoscaling", "v1", "horizontalpodautoscalers"): {
 		stub:             `{"metadata": {"name": "hpa2"}, "spec": {"maxReplicas": 3, "scaleTargetRef": {"kind": "something", "name": "cross"}}}`,
 		expectedEtcdPath: "kubernetes.io/horizontalpodautoscalers/etcdstoragepathtestnamespace/hpa2",
-		expectedGVK:      gvkP("extensions", "v1beta1", "HorizontalPodAutoscaler"), // still a beta extension
+	},
+	// --
+
+	// k8s.io/kubernetes/pkg/apis/autoscaling/v2alpha1
+	gvr("autoscaling", "v2alpha1", "horizontalpodautoscalers"): {
```

Comment:
This is `v2alpha1`, not `v1alpha1`. @DirectXMan12 what should we do, offer `v2alpha1`?